test/all.js

```diff
@@ -352,7 +352,8 @@ describe('vbot tests', async () => {
                             },{
                               type: "assertTextValue",
                               selector: ".p1",
-                              expression: "[a-z]{1,}"
+                              expression: "[a-z]{1,}",
+                              wait: 6000
```

Comment:
use the existing `waitTimeout` option instead of create a new `wait` option.
make sure the test cover this `waitTimeout` option.